src/fiskaltrust.Middleware.Interface.Client.Common/RetryLogic/RetryPolicyHandler.cs

```diff
@@ -47,5 +46,35 @@ public async Task<K> RetryFuncAsync<K>(Func<T, Task<K>> action)
 
             return default;
         }
+
+        public async Task RetryFuncAsync(Func<T, Task> action)
+        {
+            var trial = 0;
+
+            while (trial < _options.Retries)
+            {
+                try
+                {
+                    var timeoutTimer = new Timer(_options.ClientTimeout.TotalMilliseconds);
```

Comment:
Why do we use a Timer? Can´t we use the CancellationSources?

Reply:
We can and do now :D